addons/web_editor/static/lib/odoo-editor/test/spec/copyPaste.test.js

```diff
@@ -1792,15 +1792,15 @@ describe('Copy and paste', () => {
             });
             it('should paste and transform a youtube URL in a span', async () => {
                 await testEditor(BasicEditor, {
-                    contentBefore: '<p>a<span>b[x<a href="http://existing.com">546</a>x]c</span>d</p>',
+                    contentBefore: '<p>a<span odoo-test-ignore-attr>b[x<a href="http://existing.com">546</a>x]c</span>d</p>',
                     stepFunction: async editor => {
                         await pasteText(editor, 'https://www.youtube.com/watch?v=dQw4w9WgXcQ');
                         // Ensure the powerbox is active
                         window.chai.expect(editor.commandBar._active).to.be.true;
                         // Force commandBar validation on the default first choice
                         await editor.commandBar._currentValidate();
                     },
-                    contentAfter: '<p>a<span>b<iframe width="560" height="315" src="https://www.youtube.com/embed/dQw4w9WgXcQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="1"></iframe>[]c</span>d</p>',
+                    contentAfter: '<p>ab<iframe width="560" height="315" src="https://www.youtube.com/embed/dQw4w9WgXcQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen="1"></iframe>[]cd</p>',
```

Comment:
@dmo-odoo ...and here it would move it inside the iframe.

Reply:
This one is quite broken, though. I don't even know what it would do if you try to do stuff inside of the iframe.